Library/Homebrew/tap.rb

```diff
@@ -82,6 +82,12 @@ def git?
     (path/".git").exist?
   end
 
+  # The issues URL of this {Tap}.
+  # e.g. `https://github.com/user/homebrew-repo/issues`
+  def issues_url
+    "https://github.com/Homebrew/homebrew-#{repo}/issues"
```

Comment:
Yeah sorry I changed the PR after Xu’s feedback and forgot to check this again :disappointed: